src/ir/possible-contents.cpp

```diff
@@ -1165,6 +1169,10 @@ struct InfoCollector
     }
   }
   void visitRethrow(Rethrow* curr) {}
+  void visitThrowRef(ThrowRef* curr) {
+    // TODO: optimize when possible
+    addRoot(curr);
```

Comment:
I think this is unreachable, like rethrow? So there is no value to add a root here for?

Reply:
Fixed it, thanks.

Reply:
I still see `addRoot` here - maybe a change wasn't pushed?

Reply:
Wait, sorry, I had to refresh...